doc.config

```diff
@@ -0,0 +1,18 @@
+%% ./bootstrap
+%% REBAR_CONFIG=doc.config ./rebar3 ex_doc
+
+{project_plugins, [
+    {rebar3_ex_doc, "0.2.30"},
+    {rebar3_hex, "7.0.11"}
+]}.
+
+{hex, [{doc, #{provider => ex_doc}}]}.
+{ex_doc, [
+    {source_url, ~"https://example.com"},
+    {extras, [
+        ~"README.md",
+        ~"LICENSE",
+        ~"CONTRIBUTING.md"
```

Comment:
This sigil usage is generally supported starting in OTP-27, which means it is not necessarily usable for OTP-26, which is still currently supported.